the-name-card/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,21 @@
+import './Card.scss';
+import imageUrl from './../../assets/svgs/undraw_portfolio_update_re_jqnp.svg';
+
+const details = {
```

Comment:
assuming this is dummy placeholder data first right - good to add a comment just to note